lib/ukatomic/Makefile.uk

```diff
@@ -0,0 +1,5 @@
+$(eval $(call addlib_s,libukatomic,$(CONFIG_LIBUKATOMIC)))
+
+CINCLUDES-$(CONFIG_LIBUKATOMIC)	 		+= -I$(LIBUKATOMIC_BASE)/include
+ASINCLUDES-$(CONFIG_LIBUKATOMIC)	  	+= -I$(LIBUKATOMIC_BASE)/include
```

Comment:
Since `atomic.h` does not contain any directives that can be interpreted by the assembler, `ASINCLUDES` is strictly speaking not necessary.